### PR TITLE
chore: update skaffold image deps based on lts policy

### PR DESCRIPTION
**Pull request by aaron-prindle**

### CODE REVIEW

File: deploy/skaffold/Dockerfile.deps.slim [hunk 1]
```diff
@@ -17,9 +17,9 @@ ARG ARCH=amd64
 # Download kubectl
 FROM alpine:3.10 as download-kubectl
 ARG ARCH
-# Track default version installed by Google Cloud SDK: 406.0.0 moved to 1.22(.14)
+# Track default version installed by Google Cloud SDK: 406.0.0 moved to 1.23(.13)
```
Comment (renzodavid9) [inline deploy/skaffold/Dockerfile.deps.slim:20]: I think this should be 414.0.0? as in `Dockerfile.deps.lts`?

File: deploy/skaffold/Dockerfile.deps [hunk 2]
```diff
@@ -17,9 +17,9 @@ ARG ARCH=amd64
 # Download kubectl
 FROM alpine:3.10 as download-kubectl
 ARG ARCH
-# Track default version installed by Google Cloud SDK: 406.0.0 moved to 1.22(.14)
+# Track default version installed by Google Cloud SDK: 406.0.0 moved to 1.23(.13)
```
Comment (renzodavid9) [inline deploy/skaffold/Dockerfile.deps:20]: Same as previous comment

**Review by renzodavid9 — APPROVED** [review]

File: deploy/skaffold/Dockerfile.deps.slim (hunk 1 above)
Comment (aaron-prindle) [reply deploy/skaffold/Dockerfile.deps.slim]: resolved now, thanks!

File: deploy/skaffold/Dockerfile.deps (hunk 2 above)
Comment (aaron-prindle) [reply deploy/skaffold/Dockerfile.deps]: resolved now, thanks!

**Review by gsquared94 — APPROVED** [review]

**Pull request merged**